packages/components/src/focal-point-picker/test/media.tsx

```diff
@@ -7,19 +7,25 @@ import { render, screen } from '@testing-library/react';
  * Internal dependencies
  */
 import Media from '../media';
+import type { FocalPointPickerMediaProps } from '../types';
+
+const props: FocalPointPickerMediaProps = {
+	alt: '',
+	src: '',
```

Comment:
Literally every test will have the `data-testid="media"` prop, should we add this to `props` and remove that prop from the tests?

We might need to add the `data-testid` prop to `FocalPointPickerMediaProps` since that prop doesn't exist there.

Reply:
Good question - I've updated the default prop object to include `'data-testid': 'media'`

The change is only required for testing, so I opted **_not_** to change the underlying `FocalPointPickerMediaProps` type definition. 

The current implementation of `Media` uses pass-through props, which allow any prop to be passed to the child component, including `data-testid`

Reply:
Interestingly, I couldn't find any tests in the codebase where the `data-testid` prop is not repeated on each component:
- [Elevation](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/elevation/test/index.tsx) repeats `data-testid="elevation"`
- [Grid](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/grid/test/grid.tsx) repeats `data-testid="grid"`
- [Panel](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/panel/test/body.tsx) repeats `data-testid="inner-content"`

There is one example that extracts the `data-testid` value to a const, but it doesn't prevent the prop repetition on the components:
- [Icon](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/icon/test/index.tsx) repeats `data-testid={ testId }`

Reply:
Thanks for taking such a thorough look! This was a very minor detail and TBH I'm good with you've come up with 👍